index.js

```diff
@@ -0,0 +1,39 @@
+import Render from './modules/render.js';
+import { DateTime } from './luxon.min.js';
+
+Render.renderBooks();
```

Comment:
- [ ] **Required:** This line of code renders all other code beneath it from displaying the tabs and making it simulate a SPA. This is because it already rendered the whole page. Kindly comment it out.

- [ ] **Required:** When this is done, the book item doesn't work as well. Kindly fix this behaviour.